torchtext/vocab.py

```diff
@@ -33,15 +33,18 @@ class Vocab(object):
     # TODO (@mttk): Populate classs with default values of special symbols
     UNK = '<unk>'
 
-    def __init__(self, counter, max_size=None, min_freq=1, specials=['<unk>', '<pad>'],
-                 vectors=None, unk_init=None, vectors_cache=None, specials_first=True):
+    def __init__(self, counter, max_size=None, max_freq=float('inf'), min_freq=1,
+                 specials=['<unk>', '<pad>'], vectors=None, unk_init=None,
+                 vectors_cache=None, specials_first=True):
         """Create a Vocab object from a collections.Counter.
 
         Arguments:
             counter: collections.Counter object holding the frequencies of
                 each value found in the data.
             max_size: The maximum size of the vocabulary, or None for no
                 maximum. Default: None.
+            max_freq: The max frequency of tokens in the vocabulary.
+                      Values less than 1 will be set to 1. Default: inf (no filter).
```

Comment:
This may not be important. Should we also check max_freq larger than min_freq?

Reply:
Thank you for reviewing!
In the case that min_freq is larger than max_freq, should we return some error?

Reply:
Definitely. It makes sense to return an error message.